src/components/preferences/General.js

```diff
@@ -191,6 +193,42 @@ function General(props) {
                         label={t("urlImportEmailLbl")}
                     />
                 </Grid>
+                <Grid item>
+                    <Field
+                        id={buildID(
+                            baseId,
+                            ids.PERIODIC_EMAIL_NOTIFICATION_SWITCH
+                        )}
+                        component={FormSwitch}
+                        name={
+                            prefConstants.keys
+                                .ENABLE_PERIODIC_EMAIL_NOTIFICATION
+                        }
+                        color="primary"
+                        inputProps={{
+                            "aria-label": t("periodicEmailLbl"),
+                        }}
+                        label={t("periodicEmailLbl")}
+                    />
+                </Grid>
+                <Grid item xs={6}>
+                    <Field
+                        id={buildID(baseId, ids.PERIODIC_NOTIFICATION_PERIOD)}
+                        name={prefConstants.keys.PERIODIC_NOTIFICATION_PERIOD}
+                        label={t("periodicPeriodLbl")}
+                        component={FormSelectField}
+                    >
+                        <MenuItem key={1} value={3600}>
+                            1 hour
+                        </MenuItem>
+                        <MenuItem key={2} value={14400}>
+                            4 hours
+                        </MenuItem>
+                        <MenuItem key={3} value={86400}>
+                            24 hours
+                        </MenuItem>
```

Comment:
These `key` props aren't needed unless these components are nested inside an array `[]`.

Reply:
Ah, right. I copied from somewhere else that was creating an array from another array, rather than a hardcoded list like this.